modules/mod_ginger_rdf/models/m_rdf.erl

```diff
@@ -20,6 +20,8 @@
 
 m_find_value(#rdf_resource{} = Rdf, #m{value = undefined} = M, _Context) ->
     M#m{value = Rdf};
+m_find_value(Id, #m{value = undefined}, Context) when is_integer(Id) ->
+    mochijson2:encode(ginger_json_ld:serialize(to_triples(Id, Context)));
```

Comment:
But `m_value` is only triggered for `{{ m.rdf }}`, not `{{ m.rdf[id }}`.

Anyway, there shouldn’t be a problem, because of the `is_integer` guard. You 
- either pass an RDF resource (`m.rdf[id.rdf]['http://bla.com/some_predicate']`)
- or call `m.rdf[id]` with a rsc id and without any arguments, which will dump the whole thing.